app/assets/javascripts/Components/dashboard.jsx

```diff
@@ -30,16 +30,52 @@ class Dashboard extends React.Component {
             backgroundColor: 'rgb(75, 192, 192)',
           },
         ],
-      }
+      },
+      options: {},
     };
   }
 
+  getGradeEntryFormColumnBreakdown = () => {
+    // Helper function to make the AJAX request, then use its response to set the chart state
+    $.ajax({
+      url: Routes.column_breakdown_grade_entry_form_path(
+        this.state.assessment_id
+      ),
+      method: 'GET',
+      success: (data) => {
+        // Load in background colours
+        for(let j = 0; j < data["datasets"].length; j++){
```

Comment:
Use [`for ... of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) here instead of an index.